### PR TITLE
Add a separate label for SimChannel. This is needed for ProtoDUNE refactored simulation. The changes were made by Leigh Whitehead.

### PR DESCRIPTION
**Pull request by yangtj207**

### CODE REVIEW

**Comment by yangtj207** [comment]
trigger build

**Comment by lgarren** [comment]
@yangtj207  Unfortunately, we have just merged the new directory structure, which cleanly separates tensorflow, trtis, and keras, into larrecodnn.  You will need to make a new PR based on the new directory structure.

**Comment by yangtj207** [comment]
trigger build

**Comment by yangtj207** [comment]
trigger build

**Comment by yangtj207** [comment]
Hi @lgarren, I have merged the latest develop version of larreodnn into this PR. Could this PR be considered for merging? Thanks.

**Comment by lgarren** [comment]
@yangtj207  It looks good.  Thanks for all the work.  We need level 2 approval next.

**Comment by absolution1** [comment]
Looks good + CI looks good (uboone CI looks like its going through some thing but they seem unrelated).

**Comment by absolution1** [comment]
approve

**Comment by lgarren** [comment]
approve
